docker/setup.Dockerfile

```diff
@@ -13,11 +13,10 @@ RUN apk add sqlite bash --no-cache
 WORKDIR /setup
 
 # copy needed files from repo
-COPY resources/ resources/
```

Comment:
Is this PR based on some other branch?

Reply:
See https://github.com/DarkflameUniverse/DarkflameServer/pull/457#discussion_r808258121 above.